packages/nc-gui-v2/pages/index/index/create-external.vue

```diff
@@ -209,15 +265,48 @@ const testConnection = async () => {
   }
 }
 
+const handleImportURL = async () => {
+  if (!importURL.value || importURL.value === '') return
+
+  const connectionConfig = await api.utils.urlToConfig({ url: importURL.value })
```

Comment:
Could you add a `todo` that `urlToConfig` doesn't exist on `api.utils` type?

Reply:
This should be fixed by rebuilding nocodb-sdk

Reply:
Ah ok, my bad 😄